halyard-core/halyard-core.gradle

```diff
@@ -6,4 +6,5 @@ dependencies {
   compile spinnaker.dependency('googleStorage')
 
   compile 'com.beust:jcommander:1.48'
+  compile 'com.hubspot.jinjava:jinjava:2.2.3'
```

Comment:
This exists in a few places now, need to add to spinnaker-dependencies eventually.

Reply:
It's still pretty fresh in our codebase - both the artifact & (now this) is largely unproven